Formula/volta.rb

```diff
@@ -0,0 +1,27 @@
+class Volta < Formula
+  desc "Volta: The JavaScript Launcher"
+  homepage "https://volta.sh"
+  url "https://github.com/volta-cli/volta.git", :revision => "10a9bee148d2a833069d6cc27aea56af3092dc6e"
```

Comment:
Please use a tagged revision here.

Reply:
Sorry about this, but can you provide an example? I was using `:tag` earlier and [you asked for an specific revision](https://github.com/Homebrew/homebrew-core/pull/51052#discussion_r389365172), so I'm not sure how to proceed. Should I leave them both there?

Reply:
There should be a tag and a revision

Reply:
Awesome. Will update accordingly!

Reply:
Resolved on d142d754e